notebook/static/notebook/js/outputarea.js

```diff
@@ -783,6 +787,18 @@ define([
         element.append(toinsert);
         return toinsert;
     };
+    
+    var append_mathml = function (mathml, md, element) {
+        /**
+         * This method cannot do the typesetting because the mathml first has to
+         * be on the page.
+         */
+        var type = 'application/mathml+xml';
+        var toinsert = this.create_output_subarea(md, "output_mathml", type);
+        toinsert.text(mathml);
```

Comment:
@takluyver I think this should handle sanitation: https://github.com/jupyter/notebook/pull/1672/commits/a711ddb45e9540b12ade13cbe48f91a558ffd0e3